index.html

```diff
@@ -0,0 +1,184 @@
+<!-- Validated using html validator -->
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Megan Ford NSS HTML quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<!-- NOTE: all image sources are 200px wide and square- resized to 150px in html markup. V2 will resize image sources if needed.-->
```

Comment:
Bigger is fine, so long as it's not like, 1200 pixels bigger.  Resizing should be done in CSS instead of in the HTML though.